docs/reference/mapping.asciidoc

```diff
@@ -33,25 +33,42 @@ mapping values by overriding values in the mapping during the search request.
 [discrete]
 [[mapping-dynamic]]
 == Dynamic mapping
-<<dynamic-field-mapping,Dynamic mapping>> allows you to experiment with
-and explore data when you’re just getting started. {es} adds new fields
-automatically, just by indexing a document. You can add fields to the top-level
-mapping, and to inner <<object,`object`>>  and <<nested,`nested`>> fields.
+
+When you use <<dynamic-field-mapping,dynamic mapping>> {es} automatically
+attempts to detect the data type of fields in your documents. This allows 
+you to get started quickly by just adding data to an index. If you index
+additional documents with new fields, {es} will add these automatically. 
```

Comment:
unclear antecedent for `these` (these fields or these documents?)

Reply:
💯